packages/platforms/accton/x86-64/as5915-18x/platform-config/r0/src/lib/x86-64-accton-as5915-18x-r0.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+######################################################################
+#
+# platform-config for AS5915
+#
+######################################################################
+
+x86-64-accton-as5915-18x-r0:
+
+  grub:
+
+    serial: >-
+      --port=0x3f8
+      --speed=115200
+      --word=8
+      --parity=no
+      --stop=1
+
+    kernel:
+      <<: *kernel-4-14
+
+    args: >-
+      nopat
```

Comment:
Why is that set? For performance reasons?

            nopat           [X86] Disable PAT (page attribute table extension of
                            pagetables) support.

Reply:
This is set in all accton platforms.
It might lead to some error during the bootup process in other platform.
So it is suggested to set nopat to prevent from such error.

Reply:
Sorry, that is not a good idea in my opinion. Just putting it there for consistency, is not a good thing. If PAT does not lead to an error on this platform, it should be used, shouldn’t. If it’s set for a reason, the reason should be documented. (Best by adding the option in a separate commit with the commit message containing the reason.)

Reply:
I can try to remove the nopat option to see if it can work on this platform.
Do you have any guidance of how to test this functionality?

Reply:
nopat removed.